powertools-validation/src/test/resources/schema_v7.json

```diff
@@ -1,5 +1,5 @@
 {
-  "$schema": "http://json-schema.org/draft-07/schema",
+  "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
is this mandatory? then it is actually a breaking change...

Reply:
The `$schema` dialects are actually fixed values so this actually isn't the correct identifier to use. See https://json-schema.org/understanding-json-schema/reference/schema#schema. The library itself does automatically map the standard dialects to the classpath but this library does some translations to classpath as well that I opted to just migrate since I'm not sure of the intent. 
* https://github.com/networknt/json-schema-validator/blob/46da4db03c362d472521132f8bd225bc41ef7b41/src/main/java/com/networknt/schema/resource/DefaultSchemaLoader.java#L54
* https://github.com/networknt/json-schema-validator/blob/master/src/main/java/com/networknt/schema/resource/MetaSchemaMapper.java
* https://github.com/networknt/json-schema-validator/blob/46da4db03c362d472521132f8bd225bc41ef7b41/src/main/java/com/networknt/schema/JsonSchemaFactory.java#L815-L848